.github/workflows/merge-when-ready.yml

```diff
@@ -0,0 +1,30 @@
+name: Merge Bot
+
+on:
+  pull_request:
+    types:
+      - labeled # MergeWhenReady added
+      - review_request_removed # Blocking review removed
+      - review_requested
+      - unlabeled # don't merge label removed
+  pull_request_review:
+    types:
+      - dismissed # Blocking review removed
+      - submitted # Approving review added
+  status:
+
+jobs:
+  merge:
+    runs-on: ubuntu-latest
+    name: Merge when ready
+    steps:
+    - name: Integration check
+      uses: squalrus/merge-bot@v0.2.0
```

Comment:
Why "v0.2.0"? Is it intentional? It looks like the latest one is "v0.4.0"

Reply:
He, I just copy-pasted from the example... Will fix.